src/ripple/rpc/impl/RPCVersion.cpp

```diff
@@ -0,0 +1,30 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of rippled: https://github.com/ripple/rippled
+    Copyright (c) 2012, 2013 Ripple Labs Inc.
+
+    Permission to use, copy, modify, and/or distribute this software for any
+    purpose  with  or without fee is hereby granted, provided that the above
+    copyright notice and this permission notice appear in all copies.
+
+    THE  SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+    WITH  REGARD  TO  THIS  SOFTWARE  INCLUDING  ALL  IMPLIED  WARRANTIES  OF
+    MERCHANTABILITY  AND  FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+    ANY  SPECIAL ,  DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+    WHATSOEVER  RESULTING  FROM  LOSS  OF USE, DATA OR PROFITS, WHETHER IN AN
+    ACTION  OF  CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+    OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+*/
+//==============================================================================
+
+#include <ripple/rpc/RPCVersion.h>
+
+namespace ripple {
+namespace RPC {
+
+beast::SemanticVersion const firstVersion ("1.0.0");
+beast::SemanticVersion const bestVersion ("1.0.0");
+beast::SemanticVersion const lastVersion ("1.0.0");
```

Comment:
These will either be initialized before main starts, or they will be optimized out of the program by the linker.  Since they are used elsewhere, if that optimization happens (by mistake), we'll get a link time error, and we can deal with that then.

I am assuming that we are not using these in other "initialization code" that runs before main.  If my assumption is false, then this code is buggy as they may not get initialized prior to being used in other initialization code.